Extensions/pokes.css

```diff
@@ -1,73 +1,215 @@
 }
 
-#xkit-loading_pokemon {
+/*
+ * Pokémon list style 
+*/
+.xkit-pokes-lightbox {
+	position: fixed;
+	top: 0px;
+	bottom: 0px;
+	right: 0px;
+	left: 0px; 
+	z-index: 15000;
+	overflow: hidden;
+	background-color: rgba(25,25,25,.92);
+}
+.gender {
+	display: none;
+	float: right;
+	font-family: "segoe ui symbol";
```

Comment:
please please don't use fonts that don't come standard on all machines. if
you need a font, just load a webfont. its not that hard, and most browsers
are smart enough to delay loading until needed.

On Tue, Dec 29, 2015 at 3:15 PM ThePsionic <notifications@github.com> wrote:

> In Extensions/pokes.css
> <https://github.com/new-xkit/XKit/pull/891#discussion_r48570195>:
>
> > + * Pokémon list style
> > +*/
> > +.xkit-pokes-lightbox {
> > +	position: fixed;
> > +	top: 0px;
> > +	bottom: 0px;
> > +	right: 0px;
> > +	left: 0px;
> > +	z-index: 15000;
> > +	overflow: hidden;
> > +	background-color: rgba(25,25,25,.92);
> > +}
> > +.gender {
> > +	display: none;
> > +	float: right;
> > +	font-family: "segoe ui symbol";
>
> Also be aware that Segoe UI Symbol only comes standard on Windows
> machines, so unless you supply a backup font, OSX and Linux users will
> always see the default system font (also applies below).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/891/files#r48570195>.
>